node_modules/col-core/lib/server.js

```diff
@@ -61,49 +61,9 @@ var setUpServer = module.exports.setUpServer = function() {
    * using a proper multipart form request
    */
   app.use(bodyParser.urlencoded({'limit': '250kb', 'extended': true}));
+  app.use(bodyParser.json({'limit': '250kb'}));
 
-  /*!
-   * Referer-based CSRF protection. If the request is not safe (e.g., POST, DELETE) and the origin of the request (as
-   * specified by the HTTP Referer header) does not match the target host of the request (as specified by the HTTP
-   * Host header), then the request will result in a 500 error.
-   *
-   * While referer-based protection is not highly recommended due to spoofing possibilities in insecure environments,
-   * it currently offers the best trade-off between ease of use (e.g., for cURL interoperability), effort and security
-   * against CSRF attacks.
-   *
-   * Middleware that gets called earlier, can force the CSRF check to be skipped by setting `checkCSRF` on the request.
-   *
-   * If using a utility such as `curl` to POST requests to the API, you can bypass this by just setting the referer
-   * header to "/".
-   *
-   * More information about CSRF attacks: http://en.wikipedia.org/wiki/Cross-site_request_forgery
-   */
-  app.use(function(req, res, next) {
```

Comment:
Can the bookmarklet not specify a `Referer` header? Alternatively, we could skip the CSRF check if `x-bookmarklet` authentication is used?